students/ZidanLuo/lessons/lesson03/assignment/basic_operations.py

```diff
@@ -0,0 +1,73 @@
+import logging
+from customer_model import Customer
+import peewee
+
+
+def add_customer(customer_id, name, last_name, home_address, phone_number, email_address, status, credit_limit):
+    try:
+        credit_limit = float(credit_limit)
+    except ValueError as err:
+        logging.error(err)
+        raise
```

Comment:
Excellent, limiting the number of lines in the try block is key to making troubleshooting easer. :)